Resources/Locale/en-US/inventory/components/human-inventory-controller-component.ftl

```diff
@@ -1,5 +1 @@
-human-inventory-controller-component-need-uniform-to-store-in-id-slot-text = You need a uniform to store something in your ID slot!
-human-inventory-controller-component-need-uniform-to-store-in-pockets-text = You need a uniform to store something in your pockets!
-human-inventory-controller-component-too-large-text = This is too large!
-
-set-outfit-verb-get-data-text = Set Outfit
\ No newline at end of file
+set-outfit-verb-get-data-text = Set Outfit
```

Comment:
This line should probably just be moved elsewhere so this file can be removed.

I guess either the inventory component or verb system .ftl